collector/pkg/component/analyzer/cpuanalyzer/model.go

```diff
@@ -61,6 +57,9 @@ func newSegment(pid uint32, tid uint32, threadName string, startTime uint64, end
 		IndexTimestamp:  "",
 	}
 }
+func (t *TimeSegments) updateThreadName(threadName string) {
+	t.ThreadName = threadName
+}
```

Comment:
Better put this method under the definition of its receiver `TimeSegments` to improve the readability.

Reply:
ok， I have formatted the code.